### PR TITLE
Feature: desktop-version

### PR DESCRIPTION
**Pull request by ZikaZaki**
### In this pull-request I did the followings:
 - [x] Updates some of the `html` sections.
 - [x] Added `CSS` styles needed for desktop-version.
 - [x] Added all `images` needed for desktop-version.
 - [x] Make sure the website is responsive.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,527 +1,729 @@
...
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+            </div>
+            <div class="action">
+              <button>
+                See Project
+              </button>
+            </div>
+          </div>
```
Comment (Ismailco) [inline index.html:233]: - [ ] You should stick to the design as much as possible, kindly add more space for the text part and reduce the image size, so your design looks like the Figma one

|Your design|Figma design|
|-|-|
|Screenshot_20220708_053113|Screenshot_20220708_053622|

File: index.html [hunk 2]
```diff
@@ -1,527 +1,729 @@
...
-                                        <path fill-rule="evenodd" clip-rule="evenodd" d="M9.29289 5.43962C9.68342 5.0491 10.3166 5.0491 10.7071 5.43962L16.7071 11.4396C17.0976 11.8301 17.0976 12.4633 16.7071 12.8538L10.7071 18.8538C10.3166 19.2444 9.68342 19.2444 9.29289 18.8538C8.90237 18.4633 8.90237 17.8301 9.29289 17.4396L14.5858 12.1467L9.29289 6.85384C8.90237 6.46331 8.90237 5.83015 9.29289 5.43962Z" fill="#505F79"/>
-                                    </svg>
-                                </button>
-                            </div>
-                        </div>
+                      <div class="lang-list-frame">
+                        <img src="assets/img/css.png" alt="JavaScript Photo" />
+                        <h3>CSS</h3>
+                      </div>
                     </li>
```
Comment (Ismailco) [inline index.html:618]: - [ ] The language icons should be aligned horizontally, kindly fix this so your design is identical to the Figma design

|Your design|Figma design|
|-|-|
|Screenshot_20220708_053828|Screenshot_20220708_053809|

File: index.html [hunk 3]
```diff
@@ -1,527 +1,729 @@
...
+                    placeholder="Write your message here"
+                    name="user_message"
+                    id="msg"
+                    required
+                    maxlength="500"
+                  >
+                  </textarea>
+                </li>
+                <li>
+                  <button class="submit-btn" type="submit">Get in Touch</button>
```
Comment (Ismailco) [inline index.html:720]: - [ ] You can consider aligning the button to the center  of the form and reducing the form width so it looks like the provided Figma design

|Your design|Figma design|
|-|-|
|Screenshot_20220708_054238|Screenshot_20220708_054253|

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @ZikaZaki, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (rloterh) [reply index.html:618]: This issue is well resolved! ✔️

File: index.html (hunk 3 above)
Comment (rloterh) [reply index.html:720]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Zack,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉🌟 

## Highlights 👍
- Design matches figma template


<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**